Autopilot/Inc/gimbal.hpp

```diff
@@ -0,0 +1,14 @@
+/**
+ * Gimbal command processing
+ * Author: Anthony Luo (a26luo@uwaterloo.ca)
+ */
+
+#ifndef GIMBAL_HPP
+#define GIMBAL_HPP
+
+#include <stdint.h>
+
+void setPitchPercent(float pitchTarget);
+void setYawPercent(float yawTarget);
+
+#endif
```

Comment:
newline

Reply:
when's the .clang format PR getting merged in lol

Reply:
Will bring up this task after exams to the firmware team. Probably next Tuesday